build.sbt

```diff
@@ -273,7 +272,7 @@ val main = (project in file(".")).enablePlugins(play.sbt.PlayJava, play.ebean.sb
     "-Xlint:-processing"
   ),
 
-  devSettings := Seq(("config.resource", "dev.conf"), "play.server.http.port" -> "8080"),
+  devSettings := Seq(("config.resource", "dev.conf"), "play.server.http.port" -> "8089"),
```

Comment:
Lets not change this.

Reply:
Sorry, this was a testing artifact that slipped through. Will revert.